### PR TITLE
Add Twilio Apache Airflow platform

### PR DESCRIPTION
**Pull request by pragnareddye**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: INTHEWILD.md [hunk 1]
```diff
@@ -461,6 +461,7 @@ Currently, **officially** using Airflow:
 1. [Trocafone](https://www.trocafone.com/) [[@idontdomath](https://github.com/idontdomath) & [@gseva](https://github.com/gseva) & [@ordonezf](https://github.com/ordonezf) & [@PalmaLeandro](https://github.com/PalmaLeandro)]
 1. [TruFactor](https://trufactor.io/) [[@gholmes](https://github.com/gholmes) & [@angadsingh](https://github.com/angadsingh/)]
 1. [Twine Labs](https://www.twinelabs.com/) [[@ivorpeles](https://github.com/ivorpeles)]
+1. [Twilio](https://www.twilio.com/) [[@twilio](https://github.com/twilio)]
```
Comment (pankajkoti) [inline INTHEWILD.md]:
```suggestion
1. [Twilio](https://www.twilio.com/) [[@twilio](https://github.com/twilio)]
1. [Twine Labs](https://www.twinelabs.com/) [[@ivorpeles](https://github.com/ivorpeles)]
```

Twilio needs to appear before Twine Labs as per lexicographical sorting.

**Review by pankajkoti — COMMENTED** [review]
Also, could we please update the PR title to something like "Add Twilio to the list of Apache Airflow platform users" as the current description is vague.

**Comment by pragnareddye** [comment]
@pankajkoti Done

**Review by pankajastro — APPROVED** [review]

**Comment by pragnareddye** [comment]
@pankajastro I don't have permissions to merge. Would one of you merge it?

**Comment by potiuk** [comment]
@pankajastro I don't have permissions to merge. Would one of you merge it?

Sure. I am just doing it.

**Pull request merged**